zerver/migrations/0443_userpresence_new_table_schema.py

```diff
@@ -33,6 +33,11 @@ def inner_migration(apps: StateApps, schema_editor: Any) -> None:
                     suffix = "_idx" if len(infodict["columns"]) > 1 else ""
                     is_index = True
                 new_name = schema_editor._create_index_name(new_table, infodict["columns"], suffix)
+
+                if old_name == new_name:
+                    # Don't crash if these already have the right names.
+                    continue
```

Comment:
As I just also mentioned in the czo thread right now, I don't think this can even happen:

>  I don't `old_name == new_name` could even happen here, because they will always have a different prefix (`zerver_userpresence` vs `zerver_userpresenceo`/`zerver_userpresenceold` (apparently the `old` gets truncated a bit when generating a long contraint name, but that's not really a problem i think)).

Because in this migration we're just kinda manually renaming the table and all its constraints etc. to `zerver_userpresenceold`